src/app/body/board/sprint-details/sprint-details.component.html

```diff
@@ -49,24 +49,57 @@ <h5 class="pl-2" style="font-size: 18px;">Sprint No: {{currentSprintName}}</h5>
-            </div>
-            <div class="row">
-                <h6 class="card-title col-md-2 col-4" style="font-weight: bold;font-size: 14px;">Number of task: </h6>
-                <h6 class="card-text col-md-10 col-8" id="sprintStatus" style="font-size: 12px;">{{sprintData.TotalNumberOfTask}}</h6>
+                
+                <div class="col-md-6 pl-4">
+                    <div class="row py-2">
+                        <div class="col-md-3">
+                            <h6 class="card-title " style="font-weight: bold;font-size: 15px;">Start Date:
+                            </h6>
+                        </div>
+                        <div class="col-md-9">
+                            <h6 class="card-text " id="sprintStartDate" style="font-size: 13px;">
+                                {{sprintData.StartDate}}</h6>
+                        </div>
+                    </div>
+                    <div class="row py-2">
+                        <div class="col-md-3">
+                            <h6 class="card-title " style="font-weight: bold;font-size: 15px;">End Date: </h6>
+                        </div>
+                        <div class="col-md-9">
+                            <h6 class="card-text " id="sprintEndDate" style="font-size: 13px;">{{sprintData.EndDate}}
+                        </h6>
+                        </div>
+                    </div>
+                    <div class="row py-2">
+                        <div class="col-md-3">
+                            <h6 class="card-title " style="font-weight: bold;font-size: 15px;">Status: </h6>
+                        </div>
+                        <div class="col-md-9">
+                            <h6 class="card-text " id="sprintStatus" style="font-size: 13px;">{{sprintData.Status}}</h6>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-6">
+                    <div class="row py-2">
+                        <dv class="col-md-3">
+                            <h6 class="card-title " style="font-weight: bold;font-size: 15px;">Total Tasks: </h6>
+                        </dv>
+                        <div class="col-md-9">
+                            <h6 class="card-text " id="sprintStatus" style="font-size: 13px;">{{sprintData.TotalNumberOfTask}}</h6>
+                        </div>
+                    </div>
+                    <div class="row py-2">
+                        <div class="col-md-3">
+                            <h6 class="card-title " style="font-weight: bold;font-size: 15px;">Completed Tasks: </h6>
+                        </div>
+                        <div class="col-md-9">
+                            <h6 class="card-text " id="sprintStatus" style="font-size: 13px;">{{sprintData.TotalCompletedTask}}</h6>
+                        </div>
+                    </div>
+                </div>
+
```

Comment:
I think these changes are already merged, and not required, Please verify.
Pull the changes from dev-angular update your branch and then refresh it up.